ui/app/src/models/User.ts

```diff
@@ -30,7 +31,7 @@ export interface User {
 export interface EnhancedUser extends User {
   rolesBySite: LookupTable<string[]>;
   permissionsBySite: LookupTable<string[]>;
-  sites: string[];
+  sites: Array<Site>;
```

Comment:
I wonder how this ended up being an array of sites. We have sites twice in the state. This should be a lookup of site ids. Pls create a ticket for this.